Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.5
+
+RUN git clone https://github.com/TeamHG-Memex/autologin.git /opt/autologin
```

Comment:
Looks like it's possible to write a proper ``.dockerignore`` file, so I'll try this way.